go/obscuronode/enclave/rollup.go

```diff
@@ -9,14 +9,13 @@ import (
 	"github.com/obscuronet/obscuro-playground/go/obscuronode/nodecommon"
 )
 
-var GenesisRollup = NewRollup(obscurocommon.GenesisBlock, nil, common.HexToAddress("0x0"), []nodecommon.L2Tx{}, []nodecommon.Withdrawal{}, obscurocommon.GenerateNonce(), "")
+var GenesisRollup = NewRollup(obscurocommon.GenesisBlock, nil, 0, common.HexToAddress("0x0"), []nodecommon.L2Tx{}, []nodecommon.Withdrawal{}, obscurocommon.GenerateNonce(), "")
```

Comment:
Can we use a constant instead of `0`? Not sure what that represents.

Reply:
Good point 👍